chat-server/server.js

```diff
@@ -3,10 +3,11 @@ const express = require("express");
 const cors = require("cors");
 
 const app = express();
-
+app.use(express.json());
 app.use(cors());
+ const {body,validationResult} = require("express-validator");
```

Comment:
Try to put all the require packages on the top of the file, mainly for consistency and to keep the file structure.